test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -497,29 +499,16 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, Label(string(lab
 				}
 
 				testlog.Info("Rebooting the node")
-				// reboot the node, for that we change the numa policy to best-effort
-				// Note: this is used only to trigger reboot
-				policy := "best-effort"
-				// Need to make some changes to pp , causing system reboot
-				// and check if activation files is modified or deleted
-				profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
-				Expect(err).ToNot(HaveOccurred(), "Unable to fetch latest performance profile")
-				currentPolicy := profile.Spec.NUMA.TopologyPolicy
-				if *currentPolicy == "best-effort" {
-					policy = "restricted"
-				}
-				profile.Spec.NUMA = &performancev2.NUMA{
-					TopologyPolicy: &policy,
-				}
-
-				By("Updating the performance profile")
-				profiles.UpdateWithRetry(profile)
+				rebootCmd := "chroot /rootfs systemctl reboot"
+				testlog.TaggedInfof("Reboot", "Node %q: Rebooting", workerRTNode.Name)
+				_, _ = nodes.ExecCommand(ctx, workerRTNode, []string{"sh", "-c", rebootCmd})
+				testlog.Info("Node Rebooted")
 
-				By(fmt.Sprintf("Applying changes in performance profile and waiting until %s will start updating", poolName))
-				profilesupdate.WaitForTuningUpdating(ctx, profile)
+				By("Rebooted node manually and waiting for mcp to get Updating status")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
 
-				By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
-				profilesupdate.WaitForTuningUpdated(ctx, profile)
+				By("Waiting for mcp to be updated")
```

Comment:
If you changed nothing, you don't need to wait for the MCP. 
Only wait for nodes to be ready.